spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -32,11 +34,16 @@
  * register Gravitino catalogs to Spark.
  */
 public class GravitinoDriverPlugin implements DriverPlugin {
+
   private static final Logger LOG = LoggerFactory.getLogger(GravitinoDriverPlugin.class);
 
   private GravitinoCatalogManager catalogManager;
-  private static final String[] GRAVITINO_DRIVER_EXTENSIONS =
-      new String[] {IcebergSparkSessionExtensions.class.getName()};
+  private List<String> gravitinoDriverExtensions = new ArrayList<>();
+  private boolean supportsIceberg = false;
```

Comment:
Better rename to `enableIcebergSupport`.

Reply:
done